source/draw/Image.ooc

```diff
@@ -27,6 +27,11 @@ CoordinateSystem: enum {
 	YUpward = 0x02
 }
 
+TransformMethod: enum {
```

Comment:
I think it could be nice to use it with any class that implements the `Image` *interface*.